morango/sync/backends/postgres.py

```diff
@@ -37,6 +37,18 @@ def _transaction_has_savepoint(self):
                 return True
         return False
 
+    def _is_transaction_isolation_error(self, error):
+        """
+        Determine if an error is related to transaction isolation
+        :param error: An exception
+        :return: A bool whether the error is a transaction isolation error
+        """
+        # Django can wrap errors, adding it to the `__cause__` attribute
+        for e in (error, getattr(error, '__cause__', None)):
+            if "could not serialize access due to concurrent" in str(e):
```

Comment:
Could this string potentially be internationalized, and hence not get matched here, on non-EN machines/configs?

Looks like at least some Postgres errors do get localized: https://stackoverflow.com/questions/2404261/how-to-disable-postgresql-messages-translation